.github/workflows/check-amdllpc-docker.yml

```diff
@@ -15,10 +15,10 @@ jobs:
       fail-fast: false
       matrix:
         host-os:             ["ubuntu-20.04"]
-        image-template:      ["amdvlk_%s%s:nightly"]
+        image-template:      ["amdvlkadmin/amdvlk_%s%s:nightly"]
         config:              [Release]
```

Comment:
I think you should add all the build types like asan_ubsan here too.

Reply:
You mean below, in the `feature-set`?
Yes, I plan to do that, but I need to wait until this PR is merged and the base image is built once and uploaded, otherwise the asan+ubsan `check-amdllpc-docker` jobs will fail because they can’t find the base image.